src/net.cpp

```diff
@@ -1636,7 +1638,12 @@ void ThreadOpenAddedConnections()
         for (vector<CService>& vserv : lservAddressesToAdd)
         {
             CSemaphoreGrant grant(*semOutbound);
-            OpenNetworkConnection(CAddress(vserv[i % vserv.size()]), &grant);
+            OpenNetworkConnection(CAddress(vserv[i % vserv.size()], 0), &grant);
+            //=======
+            ///* We want -addnode to work even for nodes that don't provide all
+            // * wanted services, so pass in nServices=0 to CAddress. */
+            //OpenNetworkConnection(CAddress(vserv[i % vserv.size()], 0), false, &grant);
+            //>>>>>>> 15bf863219 (Don't require services in -addnode)
```

Comment:
We can probably remove the commented diff, the bool in the commented arg is only something other than false in bitcoin in a single call in `ThreadOpenConnections`, which looks to have been significantly refactored in zcash.

Reply:
Alright let's leave this comment thread open for now.  Likely we'll circle back and clean up, per your recommendation.